docs/api/js/accounts-l1-l2.md

```diff
@@ -29,7 +29,7 @@ async approveERC20(
 | -------------------- | -------------------------------------------------------------------------------- |
 | token                | The Ethereum address of the token.                                               |
 | amount               | The amount of the token to be approved.                                          |
-| overrides (optional) | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice`, etc. |
+| overrides (optional) | Ethereum transaction overrides. May be used to pass `gasLimit`, `gasPrice`, etc. You can also provide custom address of the L1 bridge to use (the bridge provided by the Matter Labs team is used by default). |
```

Comment:
zkSync team or Matter Labs team?
I prefer the first, though once we change the network name we will need to change this : 0

Reply:
I think these are synonyms :)